crates/typst/src/foundations/calc.rs

```diff
@@ -961,6 +994,18 @@ cast! {
     v: f64 => Self::Float(v),
 }
 
+impl From<i64> for Num {
+    fn from(value: i64) -> Self {
+        Self::Int(value)
+    }
+}
+
+impl From<f64> for Num {
+    fn from(value: f64) -> Self {
+        Self::Float(value)
+    }
+}
```

Comment:
Why is this need to be defined if this is already defined above in `cast!`?

Reply:
cast only implements FromValue and IntoValue. still, I think rather than having these impls (although they don't hurt), it would be nice to have an apply1 function on Num which we could use to shorten signum.